src/oas.md

```diff
@@ -169,12 +169,22 @@ The [schema](#schema) exposes two types of fields: _fixed fields_, which have a
 
 Patterned fields MUST have unique names within the containing object.
 
-In order to preserve the ability to round-trip between YAML and JSON formats, YAML version [1.2](https://yaml.org/spec/1.2/spec.html) is RECOMMENDED along with some additional constraints:
+**Note:** While APIs may be described by OpenAPI Descriptions in either YAML or JSON format, the API request and response bodies and other content are not required to be JSON or YAML.
 
-* Tags MUST be limited to those allowed by [YAML's JSON schema ruleset](https://yaml.org/spec/1.2/spec.html#id2803231), which defines a subset of the YAML syntax and is unrelated to [[JSON-Schema-2020-12|JSON Schema]].
-* Keys used in YAML maps MUST be limited to a scalar string, as defined by the [YAML Failsafe schema ruleset](https://yaml.org/spec/1.2/spec.html#id2802346).
+#### JSON and YAML Compatibility
 
-**Note:** While APIs may be described by OpenAPI Descriptions in either YAML or JSON format, the API request and response bodies and other content are not required to be JSON or YAML.
+In order to preserve the ability to round-trip between YAML and JSON formats, YAML version [1.2](https://yaml.org/spec/1.2/spec.html) is RECOMMENDED along with the additional constraints listed in [[!RFC9512]] [Section 3.4](https://www.rfc-editor.org/rfc/rfc9512.html#name-yaml-and-json).
+
+Previous versions of this specification expressed this requirement in terms of [YAML's JSON schema ruleset](https://yaml.org/spec/1.2/spec.html#id2803231), which defines a subset of the YAML syntax and is unrelated to [[JSON-Schema-2020-12|JSON Schema]].
+Despite its name, this ruleset supports features that are not part of JSON, such as distinguishing between integers and floats (see [Data Types](#data-types)) or supporting `!!float .nan` as a value.
```

Comment:
I don't know that this is necessary. This content seems fine as part of the description in the PR, in case some one wants to understand the change later, but I don't think we need it in the spec. Just my 2 cents.

Reply:
I'd be OK with taking it out.

Reply:
@mikekistler the only thing that's really happening here is that the first sentence acknowledges why there may be a compatibility weirdness where something previously within the "RECOMMENDED" is now "NOT RECOMMENDED", but maybe we don't need to make as much of a point about that?  The second sentence here is really just me over-explaining.

Reply:
@hudlow let me see if I can just word it better, so it feels less like a tangent.

Reply:
This was re-worded and all comments must be reviewed before merge so I'll do that.